.github/workflows/publish-tenant-packs.yml

```
@@ -0,0 +1,92 @@
+    timeout-minutes: 10
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11d5960a326750d5838078e36cf38b85af677262 # v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020 # v4
+        with:
+          node-version: 20
+
+      - name: Refuse a pack directory that does not exist
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2012:info:5:9: Use find instead of ls to better handle non-alphanumeric filenames

(shellcheck)